docs/readthedocs/source/doc/Orca/Overview/distributed-training-inference.md

```diff
@@ -97,6 +100,57 @@ The `data` argument in `fit` method can be a spark DataFrame, an *XShards* or a
 
 View the related [Python API doc]() for more details.
 
+**Using *spark* backend**
+
+Users can create an `Estimator` for TensorFlow 2.x using the *spark* backend as follows:
+
+```python
+def model_creator(config):
+    model = create_keras_lenet_model()
+    model.compile(**compile_args(config))
+    return model
+
+def compile_args(config):
+    if "lr" in config:
+        lr = config["lr"]
+    else:
+        lr = 1e-2
+    args = {
+        "optimizer": keras.optimizers.SGD(lr),
+        "loss": "mean_squared_error",
+        "metrics": ["mean_squared_error"]
+    }
+    return args
+
+est = Estimator.from_keras(model_creator=model_creator,
+                           config={"lr": 1e-2},
+                           workers_per_node=2,
+                           backend="spark",
+                           model_dir=model_dir)
+```
+
+The `model_creator` argument should be a function that takes a `config_args` dictionary and returns a compiled Keras model.
+The `model_dir` argument is required for *spark* backend, it should be a share filesystem path which can be accessed by executors for culster mode.  
+
+Then users can perform distributed model training and inference as follows:
+
+```python
+def train_data_creator(config, batch_size):
+    dataset = tfds.load(name="mnist", split="train")
+    dataset = dataset.map(preprocess)
+    dataset = dataset.batch(batch_size)
+    return dataset
+stats = est.fit(data=train_data_creator,
+                epochs=max_epoch,
+                steps_per_epoch=total_size // batch_size)
+predictions = est.predict(data=df,
+                          feature_cols=['image']).collect()
+```
+
+The `data` argument in `fit` method can be a spark DataFrame, an *XShards* or a *Data Creator Function* (that returns a `tf.data.Dataset`). The `data` argument in `predict` method can be a spark DataFrame or an *XShards*. See the *data-parallel processing pipeline* [page](./data-parallel-processing.md) for more details.
+
+View the related [Python API doc]() for more details.
+
 ***For more details, view the distributed TensorFlow training/inference [page]().***
```

Comment:
If "distributed TensorFlow training/inference" doc is not available now, please put todo link here just like what pytorch estimator did.